README.md

```diff
@@ -7,6 +7,33 @@
 
 > The TypeScript Definition Manager.
 
+## Deprecation Notice: Regarding TypeScript@2.0
+
+**For users doing `typings install dt~<package> --global` and receiving error**
+
+Starting from `TypeScript@2.0`, TypeScript user can acquire typings through `npm install @types/<package>`.
+
+Those typings are coming [DefinitlyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped).
+
+In the future, [redirect](https://github.com/Microsoft/types-publisher/issues/4) will be supported enabling contributors to contribute typings in their own repo as they are doing now through [`typings/registry`](https://github.com/typings/registry).
+
+Typings in are now written in external module format instead of the global format in the master branch of DT.
+
+This causes error similar to:
+
+```sh
+> typings install dt~angular --global 
+
+typings ERR! message Attempted to compile "angular" as a global module,
+but it looks like an external module. You'll need to remove the global option to continue.
+
+```
+
+To resolve this, you can either use the TypeScript@2.0 mechanism: `npm install @types/angular`, or drop the `--global` flag: `typings install dt~angular`.
+
+`typings` will still be maintained as there are use-cases which aren't yet covered by TypeScript.
+If you are using NodeJS, most likely you can use `npm install @types/<package>` instead.
```

Comment:
This line needs some help.
Would like to give user more information on when to use `typings` and when to use `@types`